miniapp/miniapp.cpp

```diff
@@ -50,6 +51,9 @@ int main(int argc, char** argv) {
     nest::mc::communication::global_policy_guard global_guard(argc, argv);
 
     try {
+        nest::mc::util::meter_manager meters;
+        meters.checkpoint("start");
```

Comment:
Do I understand that in the current implementation we always measure these timings?